src/pyansys_sphinx_theme/__init__.py

```diff
@@ -10,7 +10,9 @@
 pyansys_logo_black = os.path.join(_this_path, "static", "pyansys-logo-black-cropped.png")
 pyansys_logo_white = os.path.join(_this_path, "static", "pyansys-logo-white-cropped.png")
 ansys_favicon = os.path.join(_this_path, "static", "ansys-favicon.png")
-
+ansys_logo_1 = os.path.join(_this_path, "static", "ansys_logo_1.pdf")
+ansys_logo_2 = os.path.join(_this_path, "static", "ansys_logo_2.pdf")
+drawing = os.path.join(_this_path, "static", "drawing.pdf")
```

Comment:
Let us give these variables a more explanatory name:

```suggestion
ansys_logo_white = os.path.join(_this_path, "static", "ansys_logo_white.pdf")
ansys_logo_white_cropped = os.path.join(_this_path, "static","ansys_logo_white_cropped.pdf")
watermark = os.path.join(_this_path, "static", "watermark.pdf")
```